### ISSUE TITLE
Multiple files

### ISSUE THREAD

**Issue by ProgrammerFS** [KoushikNavuluri/Claude-API#43]
Hi, can we upload multiple files to the chat?

**Comment by KoushikNavuluri** [comment]
Currently , File Uploading seems to be not working as expected , As Claude changed many limitations for api requests.. that too will be fixed soon :) !

**Comment by merlinarer** [comment]
> Currently , File Uploading seems to be not working as expected , As Claude changed many limitations for api requests.. that too will be fixed soon :) !

```
Traceback (most recent call last):
  File "D:\proj\test.py", line 111, in <module>
    claude_api = Client(cookie)
                 ^^^^^^^^^^^^^^
  File "D:\app\miniconda3\Lib\site-packages\claude_api.py", line 12, in __init__
    self.organization_id = self.get_organization_id()
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\app\miniconda3\Lib\site-packages\claude_api.py", line 30, in get_organization_id
    response = requests.get(url, headers=headers,impersonate="chrome110")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\app\miniconda3\Lib\site-packages\curl_cffi\requests\__init__.py", line 46, in request
    return s.request(
           ^^^^^^^^^^
  File "D:\app\miniconda3\Lib\site-packages\curl_cffi\requests\session.py", line 438, in request
    raise RequestsError(e)
curl_cffi.requests.errors.RequestsError: Failed to perform, ErrCode: 35, Reason: 'BoringSSL SSL_connect: Connection was reset in connection to claude.ai:443 '
```
I met this problem using v1.0.5, do curl_cffi and vpn conflict?

**Comment by KoushikNavuluri** [comment]
Hey Guys, File handling and other issues has been fixed ,Please Update to Package == 1.0.16.

:)

**Comment by judy905** [comment]
but it seems that i still can't upload more than one attachment... already updated to 1.0.17
